gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/computer/clustering/peerpressure/PeerPressureVertexProgram.java

```diff
@@ -81,7 +81,11 @@ private PeerPressureVertexProgram() {
     }
 
     @Override
-    public void loadState(final Graph graph, final Configuration configuration) {
+    public void loadState(final Configuration configuration, final Graph... graphs) {
+        if (graphs.length != 1) {
+            throw new IllegalStateException("Must provide one graph to use, received " + graphs.length);
+        }
+        Graph graph = graphs[0];
```

Comment:
Our style is to `final` all variables that can be marked as such - it seems like this one could be.

Reply:
Done.